proto/p4/p4runtime.proto

```diff
@@ -399,11 +400,28 @@ message RegisterEntry {
   P4Data data = 3;
 }
 
+// Used to configure the digest extern only, not to stream digests or acks
+message DigestEntry {
+  uint32 digest_id = 1;
+  // a DigestList message is streamed when the following conditions are met:
+  //   - there is at least one digest ready
+  //   - the oldest digest in the list has been waiting for at least
+  //     max_timeout_ns nanoseconds or we have gathered max_list_size digests
+  //     already
+  message Config {
```

Comment:
I don't think it is too much granularity, no.  I would not be surprised if some controller developer would wish for, and use, this granularity, because of significantly different max latency desires for different uses of a Digest extern, and/or ack_timeout_ns (maybe they do not mind, or actually want, to see all duplicates for some application of a Digest extern, but want a large ack_timeout_ns for misses on a source MAC table miss Digest message).

Reply:
Thanks, I wanted to get someone else's option on that!